pkg/asset/installconfig/powervs/session.go

```diff
@@ -182,6 +185,67 @@ func (c *BxClient) ValidateAccountPermissions() error {
 	return nil
 }
 
+//ValidateDhcpService checks for existing dhcp instance for the provided PowerVS cloud instance
+func (c *BxClient) ValidateDhcpService(ctx context.Context, svcInsID string) error {
+	ctxWithTimeout, cancel := context.WithTimeout(ctx, helpers.PIGetTimeOut)
+	defer cancel()
+	res := make(chan int)
+
+	go func() error {
+		//create Power VS DHCP Client
+		dhcpClient := instance.NewIBMPIDhcpClient(ctxWithTimeout, c.PISession, svcInsID)
+		//Get all DHCP Services
+		dhcpServers, err := dhcpClient.GetAll()
+		if err != nil {
+			return err
```

Comment:
Wrap the errors that come from other APIs

Reply:
Done